lua/scratch/utils.lua

```diff
@@ -66,17 +66,36 @@ function M.getSelectedText()
 end
 
 ---Open window or set current window to new buffer
----@param abs_path string
+---@param abs_path string if win32 MUST <DISK>:\\path\to\file i dont know why
 ---@param config? vim.api.keyset.win_config
 function M.open_(abs_path, config)
   local buf = vim.api.nvim_create_buf(true, false)
-  vim.api.nvim_buf_set_name(buf, abs_path)
+  --- BUG: if buffer exist throw error, so need find existed buffer with that name
+  --- TODO: REFACTOR THAT
+  local suc = pcall(vim.api.nvim_buf_set_name, buf, abs_path)
```

Comment:
find 1 pathetic bug. When you try to create a new buffer and set name to him that already exists in buflist, an error will be thrown. 

I don't really know if it needs to be fixed or not. Because this function is called when creating a new file.